File: apps/web-react/src/lib/runtimeConfigDefaults.ts
import type { LlmRuntimeConfig, PolicySourceConfig } from '../types'

const LLM_DEFAULTS: LlmRuntimeConfig = {
  provider: 'deepseek',
  apiKey: '',
  apiKeyEnv: 'DEEPSEEK_API_KEY',
  model: 'deepseek-v4-pro',
  baseUrl: 'https://api.deepseek.com',
  responseFormat: 'json_object',
  strictJsonSchema: false,
  thinking: null,
  maxCompletionTokens: 8192,
  temperature: null,
  apiKeyConfigured: false,
  apiKeySource: 'missing'
}

const POLICY_SOURCE_DEFAULTS: PolicySourceConfig[] = [
  { name: '中国政府网', type: 'json', url: 'https://www.gov.cn/zhengce/zuixin/ZUIXINZHENGCE.json', weight: 100 },
  { name: '国家发展改革委', type: 'html', url: 'https://www.ndrc.gov.cn/xxgk/zcfb/ghwb/', weight: 92 },
  { name: '工业和信息化部', type: 'html', url: 'https://www.miit.gov.cn/zwgk/zcwj/', weight: 90 },
  { name: '科学技术部', type: 'html', url: 'https://www.most.gov.cn/xxgk/xinxifenlei/fdzdgknr/fgzc/gfxwj/', weight: 86 },
  { name: '财政部', type: 'html', url: 'https://www.mof.gov.cn/zhengwuxinxi/caizhengxinwen/', weight: 82 },
  { name: '国家能源局', type: 'html', url: 'https://www.nea.gov.cn/zcfb/', weight: 84 },
  { name: '中国证监会', type: 'html', url: 'https://www.csrc.gov.cn/csrc/c100028/zfxxgk_zdgk.shtml', weight: 80 },
  { name: '生态环境部', type: 'html', url: 'https://www.mee.gov.cn/xxgk/', weight: 84 },
  { name: '农业农村部', type: 'html', url: 'https://www.moa.gov.cn/govpublic/', weight: 82 },
  { name: '交通运输部', type: 'html', url: 'https://xxgk.mot.gov.cn/zhengceapp/740/833/list_7234.html', weight: 82 }
]

export function defaultLlmRuntimeConfig(): LlmRuntimeConfig {
  return { ...LLM_DEFAULTS }
}

export function defaultPolicySources(): PolicySourceConfig[] {
  return POLICY_SOURCE_DEFAULTS.map((source) => ({ ...source }))
}

File: apps/api/src/main/java/com/aistock/research/configuration/RuntimeConfigDefaults.java
package com.aistock.research.configuration;

import org.springframework.stereotype.Component;

import java.util.List;

@Component
public class RuntimeConfigDefaults {

    public StoredLlmConfig llm() {
        return new StoredLlmConfig(
                "deepseek",
                null,
                "DEEPSEEK_API_KEY",
                "deepseek-v4-pro",
                "https://api.deepseek.com",
                "json_object",
                false,
                null,
                8192,
                null
        );
    }

    public List<PolicySourceConfig> policySources() {
        return List.of(
                source("中国政府网", "json", "https://www.gov.cn/zhengce/zuixin/ZUIXINZHENGCE.json", 100),
                source("国家发展改革委", "html", "https://www.ndrc.gov.cn/xxgk/zcfb/ghwb/", 92),
                source("工业和信息化部", "html", "https://www.miit.gov.cn/zwgk/zcwj/", 90),
                source("科学技术部", "html", "https://www.most.gov.cn/xxgk/xinxifenlei/fdzdgknr/fgzc/gfxwj/", 86),
                source("财政部", "html", "https://www.mof.gov.cn/zhengwuxinxi/caizhengxinwen/", 82),
                source("国家能源局", "html", "https://www.nea.gov.cn/zcfb/", 84),
                source("中国证监会", "html", "https://www.csrc.gov.cn/csrc/c100028/zfxxgk_zdgk.shtml", 80),
                source("生态环境部", "html", "https://www.mee.gov.cn/xxgk/", 84),
                source("农业农村部", "html", "https://www.moa.gov.cn/govpublic/", 82),
                source("交通运输部", "html", "https://xxgk.mot.gov.cn/zhengceapp/740/833/list_7234.html", 82)
        );
    }

    private PolicySourceConfig source(String name, String type, String url, int weight) {
        return new PolicySourceConfig(name, type, url, weight);
    }
}
